main/src/ca/uwaterloo/flix/language/phase/jvm/GenContinuationInterfaces.scala

```diff
@@ -64,8 +66,28 @@ object GenContinuationInterfaces {
     // We can use `getErasedType` to map it down into one of the primitive types or to Object.
     //
 
+    // Class visitor
+    val visitor = new ClassWriter(ClassWriter.COMPUTE_FRAMES){
+      override def getCommonSuperClass(tpe1: String, tpe2: String) : String = {
+        JvmType.Obj.name.toInternalName
+      }
+    }
+
+    // Class header
+    visitor.visit(JvmOps.JavaVersion, ACC_PUBLIC + ACC_ABSTRACT + ACC_INTERFACE, interfaceType.name.toInternalName, null,
+      JvmType.Obj.name.toInternalName, null)
+
+    // `getResult()` method
+    val getResultMethod = visitor.visitMethod(ACC_PUBLIC + ACC_ABSTRACT, "getResult", s"()${resultType.toDescriptor}", null, null)
```

Comment:
So, why do we have to construct a string here with "()" why is toDescriptor not exactly the result we want, and could it be? or if not, why?

Reply:
Descriptor of a function is different than a descriptor of an object. `()` means there is no argument for the method and what comes after is the descriptor of the return type. It's basically 
```
"(" ++ args.map(_.toDescriptor).mkString ++ ")" ++ resultType.toDescriptor
```

Reply:
I wonder if we need a data type for descriptors/signatures, so we can avoid string concatenation. Maybe this is something we can discuss later today.

Reply:
We can. Shouldn't be hard.